modules/migration-changing-migration-plan-limits.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-changing-migration-plan-limits_{context}']
+= Changing migration plan limits for large migrations
+
+You can change the migration plan limits for large migrations.
```

Comment:
Can we have something around those lines:

"This tool has been pre-configured with some migration limits.   Those limits are there as safety guards to reduce risk during your migration process.  We recommend changing those limits only after they have been tested in your environment."

Reply:
I've added an "IMPORTANT" note about first testing changes in their environment, following up on our gchat.